.github/workflows/create_release_draft.yml

```
@@ -90,6 +98,7 @@ jobs:
           workflow_conclusion: success
           # Use the branch
           branch: ${{ env.BRANCH}}
+          run_id: ${{ steps.last_run_id.outputs.RUN_ID }}
```

```suggestion
          run_id: ${{ steps.last_run_id.outputs.RUN_ID }}
```
Since the previous step already takes only successful builds, the `workflow_conclusion` is not needed anymore.
Also the branch doesn't need to be defined anymore since that was only used to fetch the run ID in the first place.

---

Perfect, I updated the PR removing these redundant configurations